src/main/java/panda/command/DeleteCommand.java

```diff
@@ -0,0 +1,76 @@
+package panda.command;
+import panda.exception.PandaException;
+import panda.exception.OutOfBoundsException;
+import panda.component.*;
+public class DeleteCommand extends Command {
+    private int idx;
```

Comment:
Would it be better for "idx" to be "taskIndex" for clarity? The same applies to the rest of the similar code.